### ISSUE TITLE
is there no cls_pw in yolo11?

### ISSUE THREAD

**Issue by sheepyd** [ultralytics/ultralytics#19462]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

In yolov5,there is a hyperparameter named cls_pw,which means Classification Positive Weight.But in yolo11,I cant find this parameter.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @sheepyd, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) where you can explore details about our latest YOLO models and other hyperparameters. Many key concepts, including differences across versions, may already be clarified there.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question or a general inquiry, providing more context, such as details about what you are trying to achieve or the behavior you're observing, will help ensure an accurate and helpful response. Additionally, it's worth confirming whether you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best! Engage with our community on [Discord](https://discord.com/invite/ultralytics) 🎧 for real-time discussions, dive into technical threads on our [Discourse](https://community.ultralytics.com/), or browse our [Subreddit](https://reddit.com/r/Ultralytics) to share knowledge.

## Upgrade

Ensure that you are working with the latest version of Ultralytics to access recent features and fixes. Upgrade your `ultralytics` package along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

Our models, including YOLO, are optimized for a variety of environments. Verify you are running in an up-to-date and supported environment:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM options are available. Refer to our [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/).
- **Amazon** Deep Learning AMI instructions are documented in our [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).
- **Docker Image**: Test environments with our Docker setup (<a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>). The setup instructions are here: [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/).

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing, ensuring the correctness of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu platforms.

This is an automated response to assist you swiftly 😊. An Ultralytics engineer will also take a closer look and provide further help soon!

**Comment by Y-T-G** [comment]
It's not there in Ultralytics
